paderbox/array/interval/core.py

```diff
@@ -274,6 +273,12 @@ def __array__(self, dtype=np.bool):
         else:
             return self[:]
 
+    @staticmethod
+    def _reduce_restore(intervals_as_str, shape, inverse_mode):
+        interval = ArrayInterval.from_str(intervals_as_str, shape)
```

Comment:
Should we add the `inverse_mode` flag to `from_str`?

Reply:
Yes, we could do that